.github/workflows/unit-tests.yml

```diff
@@ -114,15 +114,27 @@ jobs:
           path: ${{ github.run_id }}
 
       - name: Run tests
+        # Tests in this stage generate code coverage metrics for the repository
+        # Coverage data is uploaded to Codecov in subsequent stages
         env:
           BIONEMO_DATA_SOURCE: ngc
         run: ./ci/scripts/run_pytest.sh --no-nbval --skip-slow
 
+      - name: Run slow tests
+        if: |
+          github.event_name == 'push' ||
```

Comment:
I dont think there will be a pull request label if the event name is push, that means essentially we're running post-merge in `main`

Reply:
werent it what we want? to run full post merge pipeline on `main` ?